libs/providers/growthbook-client/README.md

```diff
@@ -0,0 +1,43 @@
+# growthbook-client Provider
+
+## Installation
+
+```
+$ npm install @openfeature/growthbook-client-provider
+```
+
+## Example Setup
+
+```typescript
+import { GrowthBook, Context, InitOptions } from '@growthbook/growthbook';
+
+/*
+ * Configure your GrowthBook instance with GrowthBook context
+ * @see https://docs.growthbook.io/lib/js#step-1-configure-your-app
+ */
+const gbContext: Context = {
+  apiHost: 'https://cdn.growthbook.io',
+  clientKey: 'sdk-abc123',
+  // Only required if you have feature encryption enabled in GrowthBook
+  decryptionKey: 'key_abc123',
+};
+
+/*
+ * optional init options
+ * @see https://docs.growthbook.io/lib/js#switching-to-init
+ */
+const initOptions: InitOptions = {
+  timeout: 2000,
+  streaming: true,
```

Comment:
The provider will need to emit a change event for streaming changes to work. Here's an example of how it could look.

https://github.com/open-feature/js-sdk/blob/main/packages/client/src/provider/in-memory-provider/in-memory-provider.ts#L58